packages/@vue/cli-service/lib/Service.js

```diff
@@ -178,14 +178,14 @@ module.exports = class Service {
           ) {
             let apply = () => {}
             try {
-              apply = require(id)
+              apply = require(resolveModule(id, this.pkgContext))
```

Comment:
Is it better to use `loadModule` directly?  `require(resolveModule(..` looks a little redundant to me